### ISSUE TITLE
LF_ClassificationApplication shows an odd message about training.

### ISSUE THREAD

**Issue by johann-petrak** [GateNLP/gateplugin-LearningFramework#14]
The message about no documents seen and training not possible shown incorrectly:
```
Processing finished, but got an error or no documents seen, cannot train!
```

**Comment by johann-petrak** [comment]
Turns out this message really comes from a training PR which was added to the same conditional controller, but turned off. 
The problem here is that the traffic lights in a conditional pipeline only prevent the execute() invocation but not the controller started/finished/aborted invocation, so the finished callback of the disabled training PR is still invoked. The PR has no way to know if this got invoked from a disabled PR or if it really did not see any documents. 
For now we will fix the message to indicate that it comes from the disabled PR.

**Comment by johann-petrak** [comment]
Fixed by 8b127e3bbbe54e6425b549b10b10db49a0e4ea15
